aimmo-game/simulation/game_state.py

```diff
@@ -77,6 +78,16 @@ def active_avatars(self):
     def avatar_location(self, avatar_id):
         return self._avatar_manager.get_avatar(avatar_id).location
 
+    def avatar_at(self, location):
+        try:
+            return self._world_map.get_cell(location).avatar.user_id
+        except AttributeError:
+            return None
+
+    def decide_action(self, avatar_id):
```

Comment:
I thought this was going on `DecisionThread` to remove the ability to change `GameState`?